docs/conf.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+#
+# rauc-hawkbit-updater documentation build configuration file, created by
+# sphinx-quickstart on Fri Jan 22 16:00:15 2016.
```

Comment:
Hm, better generate this with `sphinx-quickstart` instead? The generated version does not contain so much commented defaults and has better author etc. handling (less duplication).

Reply:
Yes, I re-created this now. I assumed the version I copied from rauc was also the result form a `sphinx-quickstart` run, but most of the things in there we do not really need. The new version looks much shorter.